README.md

```diff
@@ -48,17 +47,20 @@ Address: 0x123456789abcdef...
 Keystore File Path: /home/user/onramp-contracts/xchain_key.json
 ```
 
-🔹 This saves the keystore file **at the exact location specified**.
-🔹 The file is **password-protected** and should be stored securely.
+🔹 This saves the keystore file **at the exact location specified**. The file is **password-protected** and should be stored securely.
+
+🔹 To run xChainClient for a specific chain, you need to request some test token to this wallet address from that chain. 
 
 ---
 
 ## 🏃‍♂️ Running the Daemon
 
+Once you have finisehd the above process and sucessfully deployed [onramp contracts](https://github.com/FIL-Builders/onramp-contracts) to the source chain and Filecoin. You should update the `config.json` with the correct information.
+
 To start the Xchain adapter daemon, run:
 
 ```sh
-./xchainClient --config ./config/config.json daemon --buffer-service --aggregation-service
+./xchainClient daemon --config ./config/config.json --chain avalanche --buffer-service --aggregation-service
```

Comment:
thanks yup! This was an annoying gotcha this is the right format for the readme!